### DISCUSSION TITLE
Isn't the default release name the tag?

### DISCUSSION THREAD

**Discussion by PaulRBerg** [anton-yurchenko/git-release#93] [Q&A] [answered]
### Description

The README currently says that the default value of `RELEASE_NAME` is the empty string (""), however I don't think that this is the case.

I've recently used `git-release` with the first example workflow suggested [here](https://github.com/anton-yurchenko/git-release/blob/main/docs/example.md#examples), and the title of the generated release was the tag itself.

For the avoidance of doubt - I like it that it uses the tag, it's just that I think the documentation for the `RELEASE_NAME` env var should be updated to reflect this behavior.

_Side note_: will you enable [discussions](https://docs.github.com/en/discussions) in this repository? I would have preferred to create a discussion rather than an issue.

### Reference

- https://github.com/PaulRBerg/prb-contracts/actions/runs/4112921082
- https://github.com/PaulRBerg/prb-contracts/releases/tag/v5.0.2

**Comment by anton-yurchenko** [comment, marked as answer]
Hello @PaulRBerg,
I have enabled discussions and migrated your issue. It was an excellent recommendation, thanks.

[git-release](https://github.com/anton-yurchenko/git-release) creates releases with the name of the tag.
You might add a text before or after the tag using `RELEASE_NAME_PREFIX` and `RELEASE_NAME_SUFFIX` options.
Alternatively, you may have full control using `RELEASE_NAME` setting.

This behavior is different when working with `UNRELEASED` option, as the tag should be updated to the HEAD.
Release will be called `Latest` in this situation. However, you can still control it using `RELEASE_NAME`.

Here the section responsible for this logic:
https://github.com/anton-yurchenko/git-release/blob/1493c1877571ab4e6ff40244789869bd50fd22c5/release/release.go#L46-L52

I hope this clarifies this confusion.

**Reply by PaulRBerg** [reply]
Yup, all clear now, thanks for taking the time to write this answer up, @anton-yurchenko.

I'm glad you like the discussions idea.
